assets/blocks/course-list-block/index.js

```diff
@@ -52,3 +53,44 @@ export const registerCourseListBlock = () => {
 		scope: [ 'inserter' ],
 	} );
 };
+
+// Hide the settings which are inherited from the Query Loop block
+// but not applicable to our Course List block.
+const withUnnecessarySettingsHidden = ( BlockEdit ) => {
+	return ( props ) => {
+		if (
+			'core/query' === props.name &&
+			props.isSelected &&
+			props.attributes &&
+			props.attributes.className &&
+			'course-list-block' === props.attributes.className
+		) {
+			setTimeout( () => {
+				const postTypeContainerQuery =
+					'.components-input-control__label:contains(' +
+					/* eslint-disable-next-line @wordpress/i18n-text-domain */
+					__( 'Post type' ) +
+					')';
+				const inheritContextContainerQuery =
+					'.components-toggle-control__label:contains(' +
+					/* eslint-disable-next-line @wordpress/i18n-text-domain */
+					__( 'Inherit query from template' ) +
+					')';
+
+				// eslint-disable-next-line no-undef
+				const toBeHiddenSettingContainers = jQuery(
```

Comment:
Can we not use jQuery, there are some movements to deprecate it, and considering that document.query selector is very powerful.

Reply:
Thanks for the review! :) I like query selector too, but .parents() and querying with inner text with query were a bit handy in jQuery. Rewrote with JS here https://github.com/Automattic/sensei/pull/5449/commits/efae9b75e00d31faf17d6543db03a919a857f3eb, can you kindly take a look again?